examples/nft-quest/abi/paymaster.ts

```diff
@@ -0,0 +1,6 @@
+import type { AddressByChain } from "./types";
+
+export const addressByChain: AddressByChain = {
```

Comment:
I think it might be better to store addresses in the same way as we do in auth-server. Less files, everything is in one place and less repetitive code